portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -583,6 +583,25 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	'getGeneticProfileIds': function () {
 	    return this.genetic_profile_ids;
 	},
+	'getDefaultGeneticProfileId': function () {
+			var gp_ids = this.getGeneticProfileIds();
+			var chosen_ids = [];
+			for (var i = 0; i < gp_ids.length; i++) {
+				if (~gp_ids[i].indexOf('Zscores') && ~gp_ids[i].indexOf('mrna')) {
```

Comment:
I think it tests if it’s unequal to `-1`, as `~(-1) == 0`. But I believe that this is not a common way to express this, and it already seems to be confusing people. I suggest rephrasing it to `gp_ids[i].indexOf('Zscores') != -1 && gp_ids[i].indexOf('mrna') != -1`, even if it is a bit more verbose.